force-app/main/default/components/ApplicationFormSectionInput.component

```diff
@@ -193,14 +193,4 @@
         </div>
     </apex:outputPanel>
 
-    <div class="slds-form-element__help">
```

Comment:
@jjheaney, did this not work. I would prefer that the assistive text be here so that in other circumstances when input is called the assistive text will be there (it currently isn't when calling up the related record dialog for example.

Reply:
The variables need to be wrapped different in this section:
`<div class="slds-form-element__help">
        {!IF(ISBLANK((questionWrapper['question']['assistiveText']), 'This field is required.', questionWrapper['question']['assistiveText'])}
    </div>`

Reply:
It did not work. I left a note here explaining the details: https://github.com/UniversityOfSaintThomas/UST-EASY/pull/101#issuecomment-1402869591

I strongly suspect that your solution will work. However, it has not been tested, and I don't think there's time for both me and Lisan to test it prior to a release tomorrow. I think we should make your suggested change in the NEXT release, but push this code (which fixes a lot of things, even if not everything) tomorrow. Anything else seems (to me) too risky to Production, but it is your call.